Card.cpp

```diff
@@ -0,0 +1,187 @@
+#include "Card.hpp"
+#include <string>
+
+int Card::getCardCode()
+{
+    return this->cardCode;

```

Comment:
nie musimy wywoływać this wewnątrz klasy.
samo return cardCode będzie  wystarczyć

Pousuwaj this i możesz sobie kliknąć merga do mastera

Reply:
Teraz jest ok, czy w setterach też usunąć?

Reply:
tez usun

Reply:
Teraz się zgadza?